flink-java/src/main/java/org/apache/flink/api/java/hadoop/mapred/HadoopOutputFormatBase.java

```diff
@@ -126,18 +128,18 @@ public void open(int taskNumber, int numTasks) throws IOException {
 			this.jobConf.setInt("mapreduce.task.partition", taskNumber + 1);
 
 			try {
-				this.context = HadoopUtils.instantiateTaskAttemptContext(this.jobConf, taskAttemptID);
+				this.context = new TaskAttemptContextImpl(this.jobConf, taskAttemptID);
 			} catch (Exception e) {
-				throw new RuntimeException(e);
+				throw new IOException("Could not create instance of TaskAttemptContext.", e);
```

Comment:
I think the `TaskAttemptContextImpl` construction does not throw any Exception, so we can drop the try/catch block around it.